godot-core/src/registry/class.rs

```diff
@@ -458,6 +467,26 @@ fn fill_class_info(item: PluginItem, c: &mut ClassRegistrationInfo) {
             .expect("duplicate: recreate_instance_func (def)");
 
             c.godot_params.is_exposed = sys::conv::bool_to_sys(!is_internal);
+            #[cfg(before_api = "4.4")]
+            let _ = icon; // mark used
+            #[cfg(since_api = "4.4")]
+            if let Some(icon_path) = icon {
+                // Convert to GString and store in global map to keep it alive for program lifetime
```

Comment:
End all the comments with a dot `.`, to make it clear where they end – it allows us to assert that given comment is present in full and part of it haven't been lost in any refactor and whatnot.
 
```suggestion
                // Convert to GString and store in global map to keep it alive for program lifetime.
```